packages/core/src/physics/CharacterController.ts

```
@@ -10,10 +10,27 @@ import { ColliderShape } from "./shape";
  * The character controllers.
  */
 export class CharacterController extends Collider {
+  // Character can overcome obstacle less than the height(stepOffset + contractOffset(0.1))
   private _stepOffset: number = 0.5;
   private _nonWalkableMode: ControllerNonWalkableMode = ControllerNonWalkableMode.PreventClimbing;
   private _upDirection = new Vector3(0, 1, 0);
+  // The default value is 0.707, which is the cosine value of 45 degrees.
   private _slopeLimit: number = 0.707;
```

Biome

[error] Prefer constants from the standard library.

Unsafe fix: Use Math.SQRT1_2 instead.


(lint/suspicious/noApproximativeNumericConstant)